modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -119,6 +122,9 @@ GrainTracker::finalize()
                                                 _unique_grain_to_ebsd_num[grain_pair.first] :
                                                 grain_pair.first,
                                                 grain_pair.second._var_idx);
+
+          auto elem_it_pair = _elemental_data_2.emplace(elem_id, std::vector<unsigned int>(_n_vars, 0));
+          elem_it_pair.first->second[grain_pair.second._var_idx] = _ebsd_reader ? _unique_grain_to_ebsd_num[grain_pair.first] : grain_pair.first;
```

Comment:
That is pretty cool - might want to put a comment with what it's doing though ;-)

Do you think that these few lines are more optimal than constructing the vector first (call it `temp_vec`) and setting these values and then calling `_elemental_data_2.emplace(elem_id, std::move(temp_vec))`?

I think that would be clearer... but maybe you had a reason you're doing it this way